src/panels/lovelace/editor/view-editor/hui-edit-view.ts

```diff
@@ -76,7 +76,9 @@ export class HuiEditView extends LitElement {
         this.viewIndex
       ];
       this._config = viewConfig;
-      this._badges = badges ? processEditorEntities(badges) : [];
+      this._badges = badges
+        ? processEditorEntities(badges.filter(Boolean))
```

Comment:
Instead of adding guards everywhere, can't we filter it when Lovelace config is loaded ?